docs/content/installation.rst

```diff
@@ -10,14 +10,22 @@ Remember -- pyGenomeTracks is available for **command line usage** as well as fo
 Requirements
 -------------
 
-* Python >=3.6
+Python dependencies:
+
+* Python >= 3.6
 * numpy >= 1.16
-* intervaltree >=2.1.0
-* pyBigWig >= 0.3.4
-* hicmatrix >= 0.14
-* pysam >= 0.8
+* intervaltree >= 2.1.0
+* pyBigWig >= 0.3.16
+* hicmatrix >= 13
+* pysam >= 0.14
 * matplotlib >= 3.1.1
-* gffutils >=0.9
+* gffutils >= 0.9
```

Comment:
matplotlib is fixed to 3.1 in the receipt and not >=, or?

Reply:
in the bioconda recipe yes. In pip I think it is >=

Reply:
no it is == so I corrected